src/lib/ext-popup.js

```diff
@@ -0,0 +1,105 @@
+import {
+  getMonitorStatus,
+  startMonitor,
+  stopMonitor,
+  openActivityLogPage,
+} from '../lib/ext-listen.js';
+
+export default class Popup {
+  constructor() {
+    this.startMonitorAllBtn = document.getElementById('startMonitorBtn');
+    this.stopMonitorAllBtn = document.getElementById('stopMonitorBtn');
+    this.openActivityLogBtn = document.getElementById('actLogPage');
+    this.monitorStatusText = document.getElementById('monitorStatus');
+    this.errorMsgText = document.getElementById('errorText');
+  }
+
+  renderErrorMsg(message) {
+    this.errorMsgText.textContent = message;
+    this.errorMsgText.style.display = 'block';
+  }
+
+  renderMonitorStartedUI() {
+    this.startMonitorAllBtn.setAttribute('disabled', 'disabled');
```

Comment:
Nit, setting the value to 'disabled' is fine, but I usually prefer to set it to `true`.